TESTING-INSTRUCTIONS.md

```diff
@@ -269,10 +282,19 @@ localStorage.setItem( 'debug', 'wc-admin:tracks' );
         -   The enable/disable button should be correctly reflected in the Woocommerce payment settings screen as well.
 
 Once everything is configured and enabled do a client test
+<<<<<<< HEAD
```

Comment:
It seems like we should delete this line 🙂

Reply:
Whooops. thanks